feature-libs/quote/components/summary/actions/quote-summary-actions.component.html

```diff
@@ -2,6 +2,17 @@
   <ng-container *ngIf="cartDetails$ | async as cartDetails">
     <ng-container *ngIf="areButtonsRendered(quoteDetails.allowedActions)">
       <section [ngClass]="isFixedPosition ? 'fixed' : 'sticky'">
```

Comment:
On mobile, the button group is sticky. When scrolling to the bottom of the page, all buttons should be visible, including the new one)
![Screenshot 2024-05-15 at 11 41 13](https://github.com/SAP/spartacus/assets/33626130/2eccb16a-615d-4017-9298-078a6ac4a608)

Reply:
The button group is sticky only until the footer area is not in view. Visibility of the buttons would depend on the size of the mobile.
This behaviour existed even before the download button was added.
![image](https://github.com/SAP/spartacus/assets/28841459/ebe421e1-00ed-4f52-aa8c-e7a999af7436)
In iPhone SE none of the buttons are visible when scrolled to the bottom

Reply:
@suprishi, when one scrolls up and the order summary is not in the viewport anymore, the buttons scroll with, namely stay in the viewport. As soon as the buttons intersect with the order summary, then they are displayed underneath of the order summary and do not continue scrolling down to the bottom of the widget.
This is an expected behaviour.

Reply:
@Larisa-Staroverova yes, am I right in understanding that the behavior is consistent w.r.t my PR as well?

Reply:
yes